src/wobbly/dependencies/search.py

```diff
@@ -0,0 +1,51 @@
+"""Dependency collecting common parameters for a job search."""
+
+from typing import Annotated
+
+from fastapi import Query
+from safir.pydantic import UtcDatetime
+from vo_models.uws.types import ExecutionPhase
+
+from ..models import JobCursor, JobSearch
+
+__all__ = ["job_search_dependency"]
+
+
+async def job_search_dependency(
```

Comment:
I'm wondering if we're missing the "LAST" query param for UWS jobs?

> /{jobs}?LAST=100 The server should list only the given number (an integer greater than 0) of most recent jobs ordered by descending creationTimes.

Reply:
This can be implemented with limit. I had originally been sticking to the UWS parameters, but they're a little weird compared to the rest of our services so I decided to align this with the pagination documentation in Safir and with Gafaelfawr (which is where that came from), and will convert from the UWS names to these names in the Wobbly client code that will go into Safir.